src/main/resources/application.properties

```diff
@@ -28,6 +28,9 @@ spring.liquibase.change-log=classpath:liquibase/changelog-postgres-master.xml
 # Swagger
 spring.mvc.pathmatch.matching-strategy=ANT_PATH_MATCHER
 springdoc.swagger-ui.path=/swagger-ui
+springdoc.swagger-ui.displayOperationId=true
```

Comment:
This shows the OpenAPI operationId in the Swagger UI.  Will be useful if the API is generated and someone wants to test or correlate to the generated API code.